db-controllers/index.js

```diff
@@ -21,14 +21,12 @@ const saveMember = (email, password, zipcode, callback) => {
 };
 
 const saveRoomAndMembers = (roomID, members, callback) => {
-  const promisedMembers = members.map((memberEmail) => {
-    return db.models.User.findOrCreate({
-      where: {
-        email: memberEmail,
-        zipcode: 78702,
-      },
-    });
-  });
+  const promisedMembers = members.map(memberEmail => db.models.User.findOrCreate({
+    where: {
+      email: memberEmail,
+      zipcode: 78702,
```

Comment:
Are we still hardcoding in the zip?

Reply:
Yeah, there's still no input in the front-end from where to grab it from. Might work on that next if no one else picks it up.